.github/CONTRIBUTING.md

```diff
@@ -65,34 +90,47 @@ reviewers. If you need to pick up changes from `develop`, you can merge
 `develop` into your branch. A reviewer might ask you to rebase a
 long-running pull request in which case force pushing is okay for that
 request. Note that squashing at the end of the review process should
-also not be done, that can be done when the pull request is [integrated
+also not be done, that will be done when the pull request is [integrated
 via GitHub](https://github.com/blog/2141-squash-your-commits).
 
 Contributing to the OpenJFX codebase
 ------------------------------------------
 
-Minimum JDK 9 is required to build OpenJFX. You must have a JDK 9 installation
+JDK 11 (at a minimum) is required to build OpenJFX. You must have a JDK 11 installation
 with the environment variable `JAVA_HOME` referencing the path to Java home for
-your JDK 9 installation. By default, tests use the same runtime as `JAVA_HOME`.
-Currently OpenJFX builds are running on JDK 9 and JDK 10.
+your JDK 11 installation. By default, tests use the same runtime as `JAVA_HOME`.
+Currently OpenJFX builds are running on JDK 11 (recommended) and JDK 12.
+
+We support development in any major Java IDE (Eclipse, IntelliJ, NetBeans). IDEs can automatically configure projects based on Gradle setup.
```

Comment:
I'm not sure we really "support" it -- it's possible, but we don't have that very well explained, so "support" might be replaced with "It is possible to develop in..."

Reply:
Fixed.